### ISSUE TITLE
1604:    attempt to call upvalue 'GetSpecialization' (a nil value)

### ISSUE THREAD

**Issue by Endraine** [ascott18/TellMeWhen#2244]
### WoW Version

Cata Classic

### TellMeWhen Version

V11.0.10

### Describe the bug

Tellmewhen wont do anything, its enabled and loaded. But it wont show up my bars in combat and i cannot access the setup settings as /tmw does nothing.

### Export Strings

```text
Date: 2024-10-30 15:56:40
ID: 2
Error occured in: Global
Count: 1
Message: ...nterface/AddOns/TellMeWhen/Components/Core/Utils.lua line 1604:
   attempt to call upvalue 'GetSpecialization' (a nil value)
Debug:
   [string "=(tail call)"]: ?
   [string "@Interface/AddOns/TellMeWhen/Components/Core/Utils.lua"]:1604:
      ...nterface/AddOns/TellMeWhen/Components/Core/Utils.lua:1603
   [string "@Interface/AddOns/TellMeWhen/Components/Core/Group.lua"]:352: ShouldUpdateIcons()
   [string "@Interface/AddOns/TellMeWhen/Components/Core/Group.lua"]:393: Setup_Conditions()
   [string "@Interface/AddOns/TellMeWhen/Components/Core/Group.lua"]:477:
      ...nterface/AddOns/TellMeWhen/Components/Core/Group.lua:425
   [string "=(tail call)"]: ?
   [string "=[C]"]: ?
   [string "safecall Dispatcher[1]"]:9:
      [string "safecall Dispatcher[1]"]:5
   [string "=(tail call)"]: ?
   [string "@Interface/AddOns/TellMeWhen/TellMeWhen.lua"]:2727: UpdateNormally()
   [string "@Interface/AddOns/TellMeWhen/TellMeWhen.lua"]:2874: Update()
   [string "@Interface/AddOns/TellMeWhen/TellMeWhen.lua"]:3016: LockToggle()
   [string "@Interface/AddOns/TellMeWhen/TellMeWhen.lua"]:3134: ?()
   [string "@Interface/AddOns/Ace3/AceConsole-3.0/AceConsole-3.0.lua"]:90: ?()
   [string "@Interface/AddOns/Blizzard_ChatFrameBase/Classic/ChatFrame.lua"]:5034:
      .../AddOns/Blizzard_ChatFrameBase/Classic/ChatFrame.lua:4980
   [string "=[C]"]: ChatEdit_ParseText()
   [string "@Interface/AddOns/Blizzard_ChatFrameBase/Classic/ChatFrame.lua"]:4686:
      .../AddOns/Blizzard_ChatFrameBase/Classic/ChatFrame.lua:4685
   [string "=[C]"]: ChatEdit_SendText()
   [string "@Interface/AddOns/Blizzard_ChatFrameBase/Classic/ChatFrame.lua"]:4722: ChatEdit_OnEnterPressed()
   [string "*ChatFrame.xml:120_OnEnterPressed"]:1:
      [string "*ChatFrame.xml:120_OnEnterPressed"]:1
Locals:

AddOns:
  Swatter, v4.4.7010 (SwimmingSeadragon)
  Ace3, vRelease-r1349
  AdvancedInterfaceOptions, v2.1.0
  AngryAssignments, vv1.15.2
  AtlasLootClassic, vv4.2.0
  AtlasLootClassicData, vv4.2.0
  AtlasLootClassicDungeonsAndRaids, vv4.2.0
  AucAdvanced, v4.4.7010 (SwimmingSeadragon)
  AucFilterBasic, v4.4.7010 (SwimmingSeadragon)
  AucStatHistogram, v4.4.7010 (SwimmingSeadragon)
  AucStatiLevel, v4.4.7010 (SwimmingSeadragon)
  AucStatPurchased, v4.4.7010 (SwimmingSeadragon)
  AucStatSimple, v4.4.7010 (SwimmingSeadragon)
  AucStatStdDev, v4.4.7010 (SwimmingSeadragon)
  AucUtilFixAH, v4.4.7010 (SwimmingSeadragon)
  Auctionator, v11.0.14
  BagBrother, v
  Bagnon, v10.0.16
  Bartender4, v4.15.1
  BeanCounter, v4.4.7010 (SwimmingSeadragon)
  BigDebuffs, vv41
  BigWigs, vv367
  ClassicLootManager, vv5.3.2
  ClassicLootManagerAlerts, vv5.3.2
  ClassicLootManagerIntegrations, vv5.3.2
  ClassicLootManagerTracker, vv5.3.2
  DBMCore, v11.0.25
  DBMStatusBarTimers, v11.0.25
  DBMVPVEM, vd7f83b8
  Details, v#Details.13132.161
  DetailsCompare2, v
  DetailsEncounterDetails, v
  DetailsRaidCheck, v
  DetailsStreamer, v
  DetailsTinyThreat, v
  DetailsVanguard, v
  DruidMacroHelper, v1.3.1
  Enchantrix, v4.4.7010 (SwimmingSeadragon)
  EnchantrixBarker, v4.4.7010 (SwimmingSeadragon)
  FojjiCore, v1.4.0
  Gargul, v7.6.3
  GladiusEx, v
  HealBot, v4.4.0.43
  HealBotData, v
  HealBotOptions, v
  HealBotTips, v
  Informant, v4.4.7010 (SwimmingSeadragon)
  LootReserve, v2024-10-03
  MerfinPlus, v1.29
  MRT, v5020
  Myslot, vv5.22.1
  NugComboBar, v11.0.3
  OmniCC, v11.0.2
  Pawn, v2.10.3
  Plater, vPlater-v595-Cata
  Postal, vv4.2.1
  Prat30, v3.9.60
  ProcTextures, v
  Quartz, v3.7.13
  RCLootCouncilClassic, v0.23.2
  ReforgeLite, vv1.9.5
  Scrap, v11.0.10
  SexyMap, vv11.0.3
  ShadowedUnitFrames, vv4.3.5-classic
  SimpleItemLevel, vv43
  SlideBar, v4.4.7010 (SwimmingSeadragon)
  Stubby, v4.4.7010 (SwimmingSeadragon)
  TacoTip, v0.4.3
  TellMeWhen, v11.0.10
  TellMeWhenOptions, v
  ThreatClassic2, v2.56
  WeakAuras, v5.17.3
  WowSimsExporter, vv2.11.7
  BlizRuntimeLib v4.4.1.57294(EU) <enUS>
  (ck=835)
```

**Comment by Endraine** [comment]
![specialisations missing tmw](https://github.com/user-attachments/assets/29426f66-427a-4af1-9b1c-b41f2c68a101)

After a couple of reloads i was able to get into settings window, but i found the cause. I have setup bars to only load based on specialization and that seems broken. See screenshot.

**Comment by ascott18** [comment]
Duplicate of #2239, fixed in 11.0.11
